core/src/main/java/org/easymock/internal/MocksControl.java

```diff
@@ -373,6 +373,24 @@ public void asStub() {
         }
     }
 
+    public IExpectationSetters<Object> atLeast(int minimalNumberOfTimes) {
+        try {
+            state.times(new Range(minimalNumberOfTimes, Integer.MAX_VALUE));
+            return this;
+        } catch (RuntimeExceptionWrapper e) {
+            throw (RuntimeException) e.getRuntimeException().fillInStackTrace();
+        }
+    }
+
+    public IExpectationSetters<Object> atMost(int maximalNumberOfTimes) {
+        try {
+            state.times(new Range(1, maximalNumberOfTimes));
```

Comment:
Let me rephrase this:
if i do `atMost(5)` but i never call the mock then the test will fail, right?
My argument is that the test should not fail and hence line 387 should read
```
state.times(new Range(0, maximalNumberOfTimes));
```